atlp-rwanda/eagles-ec-fe
# Fix : extended notification component

## What Does This PR Do?

This PR extends the existing notification component to improve real-time communication by sending notifications to sellers and buyers. The enhancements include:

- **Real-Time Notifications to Sellers:** Notify sellers when a user adds their product to a wishlist or places an order.
- **Order Status Updates:** Notify buyers when the seller updates the status of their order.

## Tasks Completed

- Integrated Socket.IO for real-time communication.
- Ensured that notifications are sent to sellers when a user adds their product to the wishlist or places an order.
- Ensured that notifications are sent to buyers when the seller updates the order status.

## How to Test

1. Register a new account or log in if you already have one at [this link](https://eagles-ec-fe-git-fix-notification-eagles-ec.vercel.app/login).
2. Add some products to your wishlist or cart and complete a payment.
3. Observe the notification icon for new notifications.
4. Click on the notification icon to view all notifications.

## Pivot Tracker ID

- #187419150

## Screenshots

<img width="1440" alt="Screenshot 2024-07-30 at 17 08 34" src="https://github.com/user-attachments/assets/921edeab-eb64-4818-b51b-fe194142b746">
